### ISSUE TITLE
Form semantics change for <option value="">

### ISSUE THREAD

**Issue by adamchainz** [wilsonzlin/minify-html#71]
Current behaviour:

```python
>>> minify_html.minify('<select><option value="">select a value</option></select>')
'<select><option>select a value</select>'
```

The `value=""` is dropped. This changes the submitted value for that option. As per [MDN docs for `value` of `<option>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/option#attr-value):

> The content of this attribute represents the value to be submitted with the form, should this option be selected. If this attribute is omitted, the value is taken from the text content of the option element.

So dropping `value=""` here causes the form to submit "select a value" instead of "".

(Originally reported downstream in https://github.com/adamchainz/django-minify-html/issues/21)

**Comment by adamchainz** [comment]
I think this will have been fixed by https://github.com/wilsonzlin/html-data/commit/8451b7dd1d3c551c744092b5ac6aeaf4f4427b62

**Comment by wilsonzlin** [comment]
Thanks for raising the issue, and yes this will be fixed in the next release cut.

**Comment by adamchainz** [comment]
Hey, I don't mean to be a pain, as I know open source is rarely the priority, but would you be able to do a release soon? The lack of this bugfix is stopping me from advocating minify-html for use in Django projects, since I think `<option value="">` is pretty common.

**Comment by spirkaa** [comment]
I need this fix too. Django complains "Select a valid choice. That choice is not one of the available choices." when empty value selected.

**Comment by wilsonzlin** [comment]
I've just released a new version `0.9.0` with the fix for `value` attributes on `<option>`. Let me know if it works on your end. It's a minor version change so there may be other changes that might be worth viewing in the changelog.
